### DISCUSSION TITLE
[BUG] Windows 11 Build 22567

### DISCUSSION THREAD

**Discussion by Yuyuupi** [valinet/ExplorerPatcher#981] [General]
In the Build 22567 which was released yesterday to the Dev Channel, the Explorer just crashloops when having ExplorerPatcher installed.

**Comment by Shomnipotence** [comment]
Yes, terrible

**Comment by valinet** [comment]
Use a debugger and identify why it crashes. I told here many times that I do not test Insider Preview builds.

**Reply by Shomnipotence** [reply]
Yes I know

**Comment by ghost** [comment]
HI :) 

I had the similar issue when upgrading, I had to reboot the computer a few times, also you could try to open taskmanager - Ctrl-Shift-Escape.

Then run : msedge.exe to download the new pre-release explorer patcher. If that doesn't work, go to safemode then uninstall explorer patcher.

Then reinstall it back in normal windows :) 

hope that helps

shawn
